src/index.js

```diff
@@ -233,5 +255,6 @@ function getClassName(parseClass) {
 
 module.exports = {
   ParseServer: ParseServer,
-  S3Adapter: S3Adapter
+  S3Adapter: S3Adapter,
```

Comment:
From the perspective of the person using `parse-server` I think this way is simpler. Not that much of a change, though, I can do it that way.